src/terraform/install.sh

```diff
@@ -56,17 +56,19 @@ receive_gpg_keys() {
     export GNUPGHOME="/tmp/tmp-gnupg"
     mkdir -p ${GNUPGHOME}
     chmod 700 ${GNUPGHOME}
-    echo -e "disable-ipv6\n${GPG_KEY_SERVERS}" > ${GNUPGHOME}/dirmngr.conf
+    echo -e "disable-ipv6\nnameserver 8.8.8.8\n${GPG_KEY_SERVERS}" > ${GNUPGHOME}/dirmngr.conf
```

Comment:
I think adding `nameserver 8.8.8.8` to `dirmngr.conf` is less secure and this Feature shouldn't always add this. Also, majority of the people seem to have the Terraform Feature working fine without adding this to the config 🤔 

@Chuxel Would appreciate your expertise on this one.

If we really want to make this change, then I'd suggest adding this after couple of retries as a last resort.

Reply:
Hmmm, ok. It seems that this problem occurs in cases where the used DNS server is not able to resolve one of the used keyserver addresses:

```bash
_pgpkey-http._tcp.keyserver.ubuntu.com
_pgpkey-https._tcp.keys.openpgp.org
_pgpkey-http._tcp.keyserver.pgp.com
_pgpkey-http._tcp.pgp.mit.edu
```

In this case adding a different nameserver as option to the `dirmngr.conf` file solves the problem.

I can verify the problem with our company DNS server, they can't resolve the names. But when I'm using the DNS of my ISP there is no problem.